modules/http/src/main/java/org/mule/module/http/internal/request/HttpResponseToMuleEvent.java

```diff
@@ -108,9 +108,18 @@ else if (responseContentType.startsWith(APPLICATION_X_WWW_FORM_URLENCODED.toLowe
         }
 
 
-        MuleMessage message = new DefaultMuleMessage(muleEvent.getMessage().getPayload(), inboundProperties,
+        DefaultMuleMessage message = new DefaultMuleMessage(muleEvent.getMessage().getPayload(), inboundProperties,
                                                      null, inboundAttachments, muleContext, muleEvent.getMessage().getDataType());
 
+
+        for (String property : inboundProperties.keySet())
```

Comment:
Right, this code is duplicated. I can move it to some static method, but moving it to the constructor will introduce the original problem again